synapse/push/mailer.py

```diff
@@ -629,6 +629,27 @@ def format_ts_filter(value, format):
     return time.strftime(format, time.localtime(value / 1000))
 
 
+# XXX: This method and the next could likely be combined in a smarter way
+@staticmethod
```

Comment:
staticmethod isn't appropriate for a top-level function?

Reply:
PyCharm lied to me...